test/DynamoCoreWpfTests/PreviewBubbleTests.cs

```diff
@@ -723,5 +869,11 @@ private void RaiseMouseLeaveNode(IInputElement nv)
 
             DispatcherUtil.DoEvents();
         }
+
+        protected ModelBase GetNode(string guid)
+        {
+            Guid id = Guid.Parse(guid);
+            return ViewModel.Model.CurrentWorkspace.GetModelInternal(id);
```

Comment:
You can directly return the node model by finding that node in CurrentWorkspace.Nodes property, instead of downcasting it to and converting it to NodeModel again.

Reply:
Done.